packages/experience-editor/src/data/fields/renderHelpers/renderConfirmActionCallback.ts

```diff
@@ -0,0 +1,12 @@
+export const renderConfirmActionCallbackLink = (
+  url: string,
+  newTab: boolean,
+) => {
+  if (newTab) {
+    return `function() { window.open("${url}", "_blank"); }`;
```

Comment:
we probably want to triple check that this doesn't open us up to XSS attacks

Reply:
We talked about this in slack, but I'm going to add this as a thinking point for the roadmap of the project. And kind of punt on it for now.